google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -103,6 +103,8 @@ public Void apply(WriteLogEntriesResponse input) {
         }
       };
   private static final ThreadLocal<Boolean> inWriteCall = new ThreadLocal<>();
+  private static final boolean FLUSH_ON_ERROR = Boolean.parseBoolean(System.getenv("FLUSH_ON_ERROR"));
```

Comment:
What happens if user has not set any environment variable?

Reply:
By Default false value there.if user does not configure value